Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java

```diff
@@ -54,6 +55,20 @@
     this.crawler = new Crawler(url);
   }
 
+  ScriptedURL(String url, int isRead, String title, String content, String repImageUrl, int wordCount) {
+    this.url = url;
+    if (isRead == 1) {
+      this.isRead = true;
+    } else {
+      this.isRead = false;
+    }
+    this.title = title;
+    this.content = content;
+    this.repImageUrl = repImageUrl;
+    this.wordCount = wordCount;
```

Comment:
I noticed about that line. But no conflict issue. So I thought lightly that issue.
OMG My old code and new code were merged...
I'll update code now.